tests/unit/poll-task-test.js

```diff
@@ -17,6 +17,8 @@ module('ember-lifeline/poll-task', function(hooks) {
   hooks.beforeEach(function() {
     this.BaseObject = EmberObject.extend({
       destroy() {
+        this._super(...arguments);
```

Comment:
Generally prefer super _after_ custom stuff when in destroy (since it is the inverse of unit/constructor). Doesn’t really matter though....